opencl[wdk]:x64-windows build failure

**Host Environment**
- OS: Windows
- Compiler: MSVC 14.27.29110 (part of MS Visual Studio 2019 Community Edition)

**To Reproduce**
Steps to reproduce the behavior:
```
./vcpkg install ffmpeg[avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,fdk-aac,ffmpeg,ffplay,ffprobe,gpl,iconv,lzma,mp3lame,nvcodec,opencl,openssl,opus,postproc,sdl2,snappy,soxr,speex,swresample,swscale,theora,vorbis,vpx,wavpack,x264,x265,zlib] --triplet=x64-windows
```

**Failure logs** 
```
D:\src\vcpkg>vcpkg install ffmpeg[avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,fdk-aac,ffmpeg,ffplay,ffprobe,gpl,iconv,lzma,mp3lame,nvcodec,opencl,openssl,opus,postproc,sdl2,snappy,soxr,speex,swresample,swscale,theora,vorbis,vpx,wavpack,x264,x265,zlib] --triplet=x64-windows
Computing installation plan...
The following packages will be built and installed:
    ffmpeg[avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,core,fdk-aac,ffmpeg,ffplay,ffprobe,gpl,iconv,lzma,mp3lame,nonfree,nvcodec,opencl,openssl,opus,postproc,sdl2,snappy,soxr,speex,swresample,swscale,theora,vorbis,vpx,wavpack,x264,x265,zlib]:x64-windows
  * opencl[core]:x64-windows
  * openssl[core]:x64-windows
  * openssl-windows[core]:x64-windows
  * opus[core]:x64-windows
  * sdl2[core]:x64-windows
  * snappy[core]:x64-windows
  * soxr[core]:x64-windows
  * speex[core]:x64-windows
  * wavpack[core]:x64-windows
  * x264[core]:x64-windows
  * x265[core]:x64-windows
  * zlib[core]:x64-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
Starting package 1/13: opencl:x64-windows
Building package opencl[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\18\18942d6ace49f2719c897fd5a55f1486fd38b7ef.zip
-- Using cached D:/src/vcpkg/downloads/KhronosGroup-OpenCL-Headers-0d5f18c6e7196863bc1557a693f1509adfcee056.tar.gz
-- Cleaning sources at D:/src/vcpkg/buildtrees/opencl/src/9adfcee056-effc37b04c.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/src/vcpkg/downloads/KhronosGroup-OpenCL-Headers-0d5f18c6e7196863bc1557a693f1509adfcee056.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/opencl/src/9adfcee056-effc37b04c.clean
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_d3d10.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_d3d11.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_dx9_media_sharing.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_dx9_media_sharing_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_egl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_ext.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_ext_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_gl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_gl_ext.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_platform.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_va_api_media_sharing_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_version.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/opencl.h
-- Using cached D:/src/vcpkg/downloads/KhronosGroup-OpenCL-CLHPP-d62a02090625655e5b2d791d6a58618b043c989c.tar.gz
-- Cleaning sources at D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/src/vcpkg/downloads/KhronosGroup-OpenCL-CLHPP-d62a02090625655e5b2d791d6a58618b043c989c.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: D:/src/vcpkg/downloads/tools/python/python-3.8.3-x64/python.exe D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean/gen_cl_hpp.py -i D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean/input_cl.hpp -o D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl.hpp
    Working Directory: D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean
    Error code: Exit code 0xc0000135



    See logs for more information:

Call Stack (most recent call first):
  ports/opencl/portfile.cmake:35 (vcpkg_execute_required_process)
  scripts/ports.cmake:79 (include)


Error: Building package opencl:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: opencl:x64-windows
  Vcpkg version: 2020.06.15-nohash
```
_(running from CMD via Windows Terminal (Microsoft Store))_

**Additional context**
Already tried as well:
```
D:\src\vcpkg>D:/src/vcpkg/downloads/tools/python/python-3.8.3-x64/python.exe D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean/gen_cl_hpp.py -i D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean/input_cl.hpp -o D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl.hpp
```

(no output - AT ALL -)

and_:_
```
D:\src\vcpkg>vcpkg install ffmpeg[avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,fdk-aac,ffmpeg,ffplay,ffprobe,gpl,iconv,lzma,mp3lame,nvcodec,opencl,openssl,opus,postproc,sdl2,snappy,soxr,speex,swresample,swscale,theora,vorbis,vpx,wavpack,x264,x265,zlib] --triplet=x64-windows --editable
_opencl_
Your feedback is important to improve Vcpkg! Please take 3 minutes to complete our survey by running: vcpkg contact --survey
Computing installation plan...
The following packages will be built and installed:
    ffmpeg[avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,core,fdk-aac,ffmpeg,ffplay,ffprobe,gpl,iconv,lzma,mp3lame,nonfree,nvcodec,opencl,openssl,opus,postproc,sdl2,snappy,soxr,speex,swresample,swscale,theora,vorbis,vpx,wavpack,x264,x265,zlib]:x64-windows
    opencl[core]:x64-windows
  * openssl[core]:x64-windows
  * openssl-windows[core]:x64-windows
  * opus[core]:x64-windows
  * sdl2[core]:x64-windows
  * snappy[core]:x64-windows
  * soxr[core]:x64-windows
  * speex[core]:x64-windows
  * wavpack[core]:x64-windows
  * x264[core]:x64-windows
  * x265[core]:x64-windows
  * zlib[core]:x64-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
Starting package 1/13: opencl:x64-windows
Building package opencl[core]:x64-windows...
-- Using cached D:/src/vcpkg/downloads/KhronosGroup-OpenCL-Headers-0d5f18c6e7196863bc1557a693f1509adfcee056.tar.gz
-- Extracting source D:/src/vcpkg/downloads/KhronosGroup-OpenCL-Headers-0d5f18c6e7196863bc1557a693f1509adfcee056.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/opencl/src/9adfcee056-effc37b04c
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_d3d10.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_d3d11.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_dx9_media_sharing.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_dx9_media_sharing_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_egl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_ext.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_ext_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_gl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_gl_ext.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_platform.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_va_api_media_sharing_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_version.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/opencl.h
-- Using cached D:/src/vcpkg/downloads/KhronosGroup-OpenCL-CLHPP-d62a02090625655e5b2d791d6a58618b043c989c.tar.gz
-- Extracting source D:/src/vcpkg/downloads/KhronosGroup-OpenCL-CLHPP-d62a02090625655e5b2d791d6a58618b043c989c.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: D:/src/vcpkg/downloads/tools/python/python-3.8.3-x64/python.exe D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1/gen_cl_hpp.py -i D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1/input_cl.hpp -o D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl.hpp
    Working Directory: D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1
    Error code: Exit code 0xc0000135



    See logs for more information:

Call Stack (most recent call first):
  ports/opencl/portfile.cmake:35 (vcpkg_execute_required_process)
  scripts/ports.cmake:79 (include)


Error: Building package opencl:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: opencl:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
```

Still, not working. And I even zipped up a backup of that Python folder, by the way, too:
`D:\src\vcpkg>vcpkg install opencl[wdk] --triplet=x64-windows
Computing installation plan...
The following packages will be built and installed:
    opencl[core,wdk]:x64-windows
Detecting compiler hash for triplet x64-windows...
Starting package 1/1: opencl:x64-windows
Building package opencl[core,wdk]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\b3\b32165e610a409129c85ef9d5fa9c86521c08ecd.zip
-- Using cached D:/src/vcpkg/downloads/KhronosGroup-OpenCL-Headers-0d5f18c6e7196863bc1557a693f1509adfcee056.tar.gz
-- Cleaning sources at D:/src/vcpkg/buildtrees/opencl/src/9adfcee056-effc37b04c.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/src/vcpkg/downloads/KhronosGroup-OpenCL-Headers-0d5f18c6e7196863bc1557a693f1509adfcee056.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/opencl/src/9adfcee056-effc37b04c.clean
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_d3d10.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_d3d11.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_dx9_media_sharing.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_dx9_media_sharing_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_egl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_ext.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_ext_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_gl.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_gl_ext.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_platform.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_va_api_media_sharing_intel.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_version.h
-- Installing: D:/src/vcpkg/packages/opencl_x64-windows/include/CL/opencl.h
-- Using cached D:/src/vcpkg/downloads/KhronosGroup-OpenCL-CLHPP-d62a02090625655e5b2d791d6a58618b043c989c.tar.gz
-- Cleaning sources at D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/src/vcpkg/downloads/KhronosGroup-OpenCL-CLHPP-d62a02090625655e5b2d791d6a58618b043c989c.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean
-- Using cached D:/src/vcpkg/downloads/python-3.8.3-embed-amd64.zip
CMake Error: Problem with archive_write_finish_entry(): Can't restore time
CMake Error: Problem extracting tar: D:/src/vcpkg/downloads/python-3.8.3-embed-amd64.zip
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: D:/src/vcpkg/downloads/tools/python/python-3.8.3-x64/python.exe D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean/gen_cl_hpp.py -i D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean/input_cl.hpp -o D:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl.hpp
    Working Directory: D:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean
    Error code: Exit code 0xc0000135



    See logs for more information:

Call Stack (most recent call first):
  ports/opencl/portfile.cmake:35 (vcpkg_execute_required_process)
  scripts/ports.cmake:79 (include)


Error: Building package opencl:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: opencl:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.

D:\src\vcpkg>`

**BY THE WAY:** For some reason, vcpkg fails to extract **ANY** non-.tar.gz package at all; not**_ _**only Python itself, but also ANY source code that's zipped, or else.

Example_s_:

`D:\src\vcpkg>vcpkg install x264 tap-windows6 zfp[all] --triplet=x64-windows
Computing installation plan...
The following packages will be built and installed:
    tap-windows6[core]:x64-windows
    x264[core]:x64-windows
    zfp[all,core]:x64-windows
Detecting compiler hash for triplet x64-windows...
Starting package 1/3: tap-windows6:x64-windows
Building package tap-windows6[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\fd\fd9956e13841ccbe56a347f95e70c17a00382b47.zip
-- Downloading https://github.com/OpenVPN/tap-windows6/archive/0e30f5c13b3c7b0bdd60da915350f653e4c14d92.tar.gz...
-- Extracting source D:/src/vcpkg/downloads/OpenVPN-tap-windows6-0e30f5c13b3c7b0bdd60da915350f653e4c14d92.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/tap-windows6/src/53e4c14d92-ec156f1cb1.clean
-- Installing: D:/src/vcpkg/packages/tap-windows6_x64-windows/share/tap-windows6/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\joao2\AppData\Local\vcpkg\archives\fd\fd9956e13841ccbe56a347f95e70c17a00382b47.zip
Building package tap-windows6[core]:x64-windows... done
Installing package tap-windows6[core]:x64-windows...
Installing package tap-windows6[core]:x64-windows... done
Elapsed time for package tap-windows6:x64-windows: 4.287 s
Starting package 2/3: x264:x64-windows
Building package x264[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\a2\a2a4c098fd72220f60b1c705e6f613035e2dbf79.zip
-- Using cached D:/src/vcpkg/downloads/mirror-x264-303c484ec828ed0d8bfe743500e70314d026c3bd.tar.gz
-- Cleaning sources at D:/src/vcpkg/buildtrees/x264/src/14d026c3bd-bca2ebf740.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/src/vcpkg/downloads/mirror-x264-303c484ec828ed0d8bfe743500e70314d026c3bd.tar.gz
-- Applying patch uwp-cflags.patch
-- Using source at D:/src/vcpkg/buildtrees/x264/src/14d026c3bd-bca2ebf740.clean
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: C:/ProgramData/chocolatey/lib/cmake.portable/tools/cmake-3.18.4-win64-x64/bin/cmake.exe -E tar xzf D:/src/vcpkg/downloads/msys-binutils-2.34-4-x86_64.pkg.tar.zst
    Working Directory: D:/src/vcpkg/downloads/tools/msys2/77f3fd5f29257d6b.tmp
    Error code: 1
    See logs for more information:
      D:\src\vcpkg\buildtrees\x264\msys-x64-windows-3-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:318 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_configure_make.cmake:226 (vcpkg_acquire_msys)
  ports/x264/portfile.cmake:36 (vcpkg_configure_make)
  scripts/ports.cmake:79 (include)


Error: Building package x264:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: x264:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
`

and:

`D:\src\vcpkg>vcpkg install arrow[core,csv,filesystem,json,parquet] gamma memorymodule[unicode] mmloader[shellcode] pmdk robin-hood-hashing gmmlib --triplet=x64-windows
Computing installation plan...
The following packages will be built and installed:
    arrow[core,csv,filesystem,json,parquet]:x64-windows
  * boost-scope-exit[core]:x64-windows
  * flatbuffers[core]:x64-windows
    gamma[core]:x64-windows
  * gflags[core]:x64-windows
  * glog[core]:x64-windows
    gmmlib[core]:x64-windows
  * libevent[core,thread]:x64-windows
  * libsndfile[core,external-libs]:x64-windows
    memorymodule[core,unicode]:x64-windows
    mmloader[core,shellcode]:x64-windows
    pmdk[core]:x64-windows
  * portaudio[core]:x64-windows
    robin-hood-hashing[core]:x64-windows
  * snappy[core]:x64-windows
  * thrift[core]:x64-windows
  * uriparser[core]:x64-windows
  * utf8proc[core]:x64-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
Starting package 1/18: flatbuffers:x64-windows
Building package flatbuffers[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\ba\bac6288c74150f3ebde458e1d5434cf90d3399e8.zip
-- Note: flatbuffers only supports static library linkage. Building static library.
-- Downloading https://github.com/google/flatbuffers/archive/v1.12.0.tar.gz...
-- Extracting source D:/src/vcpkg/downloads/google-flatbuffers-v1.12.0.tar.gz
-- Applying patch ignore_use_of_cmake_toolchain_file.patch
-- Applying patch no-werror.patch
-- Applying patch fix-uwp-build.patch
-- Using source at D:/src/vcpkg/buildtrees/flatbuffers/src/v1.12.0-f85afa3f58.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: D:/src/vcpkg/packages/flatbuffers_x64-windows/share/flatbuffers/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\joao2\AppData\Local\vcpkg\archives\ba\bac6288c74150f3ebde458e1d5434cf90d3399e8.zip
Building package flatbuffers[core]:x64-windows... done
Installing package flatbuffers[core]:x64-windows...
Installing package flatbuffers[core]:x64-windows... done
Elapsed time for package flatbuffers:x64-windows: 3.985 min
Starting package 2/18: gflags:x64-windows
Building package gflags[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\a8\a8e3c60b9bd00831918492ee992b038c03a4f705.zip
-- Using cached D:/src/vcpkg/downloads/gflags-gflags-v2.2.2.tar.gz
-- Cleaning sources at D:/src/vcpkg/buildtrees/gflags/src/v2.2.2-d9283b9285.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/src/vcpkg/downloads/gflags-gflags-v2.2.2.tar.gz
-- Applying patch 0001-patch-dir.patch
-- Applying patch fix_cmake_config.patch
-- Using source at D:/src/vcpkg/buildtrees/gflags/src/v2.2.2-d9283b9285.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: D:/src/vcpkg/packages/gflags_x64-windows/share/gflags/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\joao2\AppData\Local\vcpkg\archives\a8\a8e3c60b9bd00831918492ee992b038c03a4f705.zip
Building package gflags[core]:x64-windows... done
Installing package gflags[core]:x64-windows...
Installing package gflags[core]:x64-windows... done
Elapsed time for package gflags:x64-windows: 40.99 s
Starting package 3/18: glog:x64-windows
Building package glog[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\49\49accca0de8fb5b15c0893b87fcff9129f95dbae.zip
-- Using cached D:/src/vcpkg/downloads/google-glog-v0.4.0.tar.gz
-- Cleaning sources at D:/src/vcpkg/buildtrees/glog/src/v0.4.0-46ccbc49a4.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/src/vcpkg/downloads/google-glog-v0.4.0.tar.gz
-- Applying patch glog_disable_debug_postfix.patch
-- Using source at D:/src/vcpkg/buildtrees/glog/src/v0.4.0-46ccbc49a4.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: D:/src/vcpkg/packages/glog_x64-windows/share/glog/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\joao2\AppData\Local\vcpkg\archives\49\49accca0de8fb5b15c0893b87fcff9129f95dbae.zip
Building package glog[core]:x64-windows... done
Installing package glog[core]:x64-windows...
Installing package glog[core]:x64-windows... done
Elapsed time for package glog:x64-windows: 3.036 min
Starting package 4/18: snappy:x64-windows
Building package snappy[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\2c\2c6ebcf6c077febde109b8ef325802fc6672150d.zip
-- Downloading https://github.com/google/snappy/archive/537f4ad6240e586970fe554614542e9717df7902.tar.gz...
-- Extracting source D:/src/vcpkg/downloads/google-snappy-537f4ad6240e586970fe554614542e9717df7902.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/snappy/src/9717df7902-acae3bca74.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\joao2\AppData\Local\vcpkg\archives\2c\2c6ebcf6c077febde109b8ef325802fc6672150d.zip
Building package snappy[core]:x64-windows... done
Installing package snappy[core]:x64-windows...
Installing package snappy[core]:x64-windows... done
Elapsed time for package snappy:x64-windows: 1.522 min
Starting package 5/18: boost-scope-exit:x64-windows
Building package boost-scope-exit[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\f8\f8bdbbfa24b85fde10a7740cdf3390c6aaaced14.zip
-- Downloading https://github.com/boostorg/scope_exit/archive/boost-1.73.0.tar.gz...
-- Extracting source D:/src/vcpkg/downloads/boostorg-scope_exit-boost-1.73.0.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/boost-scope-exit/src/ost-1.73.0-1ea09a00c4.clean
-- Packaging headers
-- Packaging headers done
-- Using cached D:/src/vcpkg/downloads/boost_LICENSE_1_0.txt
-- Installing: D:/src/vcpkg/packages/boost-scope-exit_x64-windows/share/boost-scope-exit/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\joao2\AppData\Local\vcpkg\archives\f8\f8bdbbfa24b85fde10a7740cdf3390c6aaaced14.zip
Building package boost-scope-exit[core]:x64-windows... done
Installing package boost-scope-exit[core]:x64-windows...
Installing package boost-scope-exit[core]:x64-windows... done
Elapsed time for package boost-scope-exit:x64-windows: 4.686 s
Starting package 6/18: libevent:x64-windows
Building package libevent[core,thread]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\ae\ae834ff983ab691389c6156ef4898e002d4a9f4c.zip
-- Using cached D:/src/vcpkg/downloads/libevent-libevent-release-2.1.12-stable.tar.gz
-- Cleaning sources at D:/src/vcpkg/buildtrees/libevent/src/.12-stable-79b01d8892.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/src/vcpkg/downloads/libevent-libevent-release-2.1.12-stable.tar.gz
-- Applying patch fix-file_path.patch
-- Applying patch fix-LibeventConfig_cmake_in_path.patch
-- Using source at D:/src/vcpkg/buildtrees/libevent/src/.12-stable-79b01d8892.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: D:/src/vcpkg/packages/libevent_x64-windows/share/libevent/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\joao2\AppData\Local\vcpkg\archives\ae\ae834ff983ab691389c6156ef4898e002d4a9f4c.zip
Building package libevent[core,thread]:x64-windows... done
Installing package libevent[core,thread]:x64-windows...
Installing package libevent[core,thread]:x64-windows... done
Elapsed time for package libevent:x64-windows: 8.045 min
Starting package 7/18: thrift:x64-windows
Building package thrift[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\22\227149760969c289ebfa168d2023447eca3c816a.zip
-- Note: thrift only supports static library linkage. Building static library.
-- Trying auto-select mirror...
-- Using cached D:/src/vcpkg/downloads/winflexbison-2.5.16.zip
-- Skipping hash check for D:/src/vcpkg/downloads/winflexbison-2.5.16.zip.
-- The default mirror is in Disaster Recovery mode, trying other mirrors...
-- Trying mirror cfhcable...
-- Using cached D:/src/vcpkg/downloads/winflexbison-2.5.16.zip
-- Skipping hash check for D:/src/vcpkg/downloads/winflexbison-2.5.16.zip.
-- Extracting source D:/src/vcpkg/downloads/winflexbison-2.5.16.zip
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: C:/ProgramData/chocolatey/lib/cmake.portable/tools/cmake-3.18.4-win64-x64/bin/cmake.exe -E tar xjf D:/src/vcpkg/downloads/winflexbison-2.5.16.zip
    Working Directory: D:/src/vcpkg/downloads/tools/winflexbison/0a14154bff-a8cf65db07.tmp
    Error code: 1
    See logs for more information:
      D:\src\vcpkg\buildtrees\thrift\extract-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_extract_source_archive.cmake:43 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_extract_source_archive_ex.cmake:115 (vcpkg_extract_source_archive)
  scripts/cmake/vcpkg_from_sourceforge.cmake:226 (vcpkg_extract_source_archive_ex)
  scripts/cmake/vcpkg_find_acquire_program.cmake:531 (vcpkg_from_sourceforge)
  ports/thrift/portfile.cmake:7 (vcpkg_find_acquire_program)
  scripts/ports.cmake:79 (include)


Error: Building package thrift:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: thrift:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.

D:\src\vcpkg>vcpkg install gamma memorymodule[unicode] mmloader[shellcode] pmdk robin-hood-hashing gmmlib --triplet=x64-windows
Computing installation plan...
The following packages will be built and installed:
    gamma[core]:x64-windows
    gmmlib[core]:x64-windows
  * libsndfile[core,external-libs]:x64-windows
    memorymodule[core,unicode]:x64-windows
    mmloader[core,shellcode]:x64-windows
    pmdk[core]:x64-windows
  * portaudio[core]:x64-windows
    robin-hood-hashing[core]:x64-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
Starting package 1/8: libsndfile:x64-windows
Building package libsndfile[core,external-libs]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\c3\c35a26519a01f0efc6e9986a7b8e3bd06f419bea.zip
-- Downloading https://github.com/libsndfile/libsndfile/archive/68958f9c9199dad97dcd10700df3746b2cd9b8a3.tar.gz...
-- Extracting source D:/src/vcpkg/downloads/libsndfile-libsndfile-68958f9c9199dad97dcd10700df3746b2cd9b8a3.tar.gz
-- Using source at D:/src/vcpkg/buildtrees/libsndfile/src/6b2cd9b8a3-da1fed769f.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:141 (message):
    Command failed: C:/ProgramData/chocolatey/lib/cmake.portable/tools/cmake-3.18.4-win64-x64/bin/cmake.exe --build . --config Debug --target install -- -v -j5
    Working Directory: D:/src/vcpkg/buildtrees/libsndfile/x64-windows-dbg
    See logs for more information:
      D:\src\vcpkg\buildtrees\libsndfile\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:92 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/libsndfile/portfile.cmake:31 (vcpkg_install_cmake)
  scripts/ports.cmake:79 (include)


Error: Building package libsndfile:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: libsndfile:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.

D:\src\vcpkg>vcpkg install gamma --triplet=x64-windows
Computing installation plan...
The following packages will be built and installed:
    gamma[core]:x64-windows
  * libsndfile[core,external-libs]:x64-windows
  * portaudio[core]:x64-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...

D:\src\vcpkg>vcpkg install arrow[core,csv,filesystem,json,parquet] memorymodule[unicode] mmloader[shellcode] pmdk robin-hood-hashing gmmlib --triplet=x64-windows
Computing installation plan...
The following packages will be built and installed:
    arrow[core,csv,filesystem,json,parquet]:x64-windows
    gmmlib[core]:x64-windows
    memorymodule[core,unicode]:x64-windows
    mmloader[core,shellcode]:x64-windows
    pmdk[core]:x64-windows
    robin-hood-hashing[core]:x64-windows
  * thrift[core]:x64-windows
  * uriparser[core]:x64-windows
  * utf8proc[core]:x64-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
Starting package 1/9: thrift:x64-windows
Building package thrift[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\22\227149760969c289ebfa168d2023447eca3c816a.zip
-- Note: thrift only supports static library linkage. Building static library.
-- Trying auto-select mirror...
-- Using cached D:/src/vcpkg/downloads/winflexbison-2.5.16.zip
-- Skipping hash check for D:/src/vcpkg/downloads/winflexbison-2.5.16.zip.
-- The default mirror is in Disaster Recovery mode, trying other mirrors...
-- Trying mirror cfhcable...
-- Using cached D:/src/vcpkg/downloads/winflexbison-2.5.16.zip
-- Skipping hash check for D:/src/vcpkg/downloads/winflexbison-2.5.16.zip.
-- Extracting source D:/src/vcpkg/downloads/winflexbison-2.5.16.zip
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: C:/ProgramData/chocolatey/lib/cmake.portable/tools/cmake-3.18.4-win64-x64/bin/cmake.exe -E tar xjf D:/src/vcpkg/downloads/winflexbison-2.5.16.zip
    Working Directory: D:/src/vcpkg/downloads/tools/winflexbison/0a14154bff-a8cf65db07.tmp
    Error code: 1
    See logs for more information:
      D:\src\vcpkg\buildtrees\thrift\extract-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_extract_source_archive.cmake:43 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_extract_source_archive_ex.cmake:115 (vcpkg_extract_source_archive)
  scripts/cmake/vcpkg_from_sourceforge.cmake:226 (vcpkg_extract_source_archive_ex)
  scripts/cmake/vcpkg_find_acquire_program.cmake:531 (vcpkg_from_sourceforge)
  ports/thrift/portfile.cmake:7 (vcpkg_find_acquire_program)
  scripts/ports.cmake:79 (include)


Error: Building package thrift:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: thrift:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
`

Still effed-up:
` D:\src\vcpkg>vcpkg install xsimd[xcomplex] --triplet=x64-windows
Computing installation plan...
The following packages will be built and installed:
  * nlohmann-json[core]:x64-windows
    xsimd[core,xcomplex]:x64-windows
  * xtl[core]:x64-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
Starting package 1/3: nlohmann-json:x64-windows
Building package nlohmann-json[core]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\21\21c29dc75dfdacff15383010c41efbf85902fefc.zip
-- Using cached D:/src/vcpkg/downloads/nlohmann-json-v3.9.1/CMakeLists.txt
-- Using cached D:/src/vcpkg/downloads/nlohmann-json-v3.9.1/LICENSE.MIT
-- Using cached D:/src/vcpkg/downloads/nlohmann-json-v3.9.1/nlohmann_json.natvis
-- Using cached D:/src/vcpkg/downloads/nlohmann-json-v3.9.1/cmake/config.cmake.in
-- Using cached D:/src/vcpkg/downloads/nlohmann-json-v3.9.1/cmake/pkg-config.pc.in
-- Using cached D:/src/vcpkg/downloads/nlohmann-json-v3.9.1/cmake/nlohmann_jsonConfigVersion.cmake.in
-- Using cached D:/src/vcpkg/downloads/nlohmann-json-v3.9.1-include.zip
-- Extracting source D:/src/vcpkg/downloads/nlohmann-json-v3.9.1-include.zip
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:85 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" -E tar xjf D:/src/vcpkg/downloads/nlohmann-json-v3.9.1-include.zip
    Working Directory: D:/src/vcpkg/buildtrees/nlohmann-json/3.9.1-74ced61d52.tmp
    Error code: 1
    See logs for more information:
      D:\src\vcpkg\buildtrees\nlohmann-json\extract-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_extract_source_archive.cmake:43 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_extract_source_archive_ex.cmake:115 (vcpkg_extract_source_archive)
  ports/nlohmann-json/portfile.cmake:28 (vcpkg_extract_source_archive_ex)
  scripts/ports.cmake:79 (include)


Error: Building package nlohmann-json:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: nlohmann-json:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above. `

Log file:

D:\src\vcpkg>cat D:\src\vcpkg\buildtrees\nlohmann-json\extract-err.log
**CMake Error: Problem with archive_write_finish_entry(): Can't restore time
CMake Error: Problem extracting tar: D:/src/vcpkg/downloads/nlohmann-json-v3.9.1-include.zip**

It was a FAT32 drive, indeed, so I moved to C:. But it still gives me an error:

C:\src\vcpkg>vcpkg install opencl[wdk]:x64-windows --x-json --x-xunit=OpenCL[WKD]
Computing installation plan...
The following packages will be built and installed:
    opencl[core,wdk]:x64-windows
Detecting compiler hash for triplet x64-windows...
Starting package 1/1: opencl:x64-windows
Building package opencl[core,wdk]:x64-windows...
Could not locate cached archive: C:\Users\joao2\AppData\Local\vcpkg\archives\9b\9bdb0e7510407c9438a38ee2d2ff49f6917085d3.zip
-- Downloading https://github.com/KhronosGroup/OpenCL-Headers/archive/0d5f18c6e7196863bc1557a693f1509adfcee056.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/KhronosGroup-OpenCL-Headers-0d5f18c6e7196863bc1557a693f1509adfcee056.tar.gz
-- Using source at C:/src/vcpkg/buildtrees/opencl/src/9adfcee056-effc37b04c.clean
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_d3d10.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_d3d11.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_dx9_media_sharing.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_dx9_media_sharing_intel.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_egl.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_ext.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_ext_intel.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_gl.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_gl_ext.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_platform.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_va_api_media_sharing_intel.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/cl_version.h
-- Installing: C:/src/vcpkg/packages/opencl_x64-windows/include/CL/opencl.h
-- Downloading https://github.com/KhronosGroup/OpenCL-CLHPP/archive/d62a02090625655e5b2d791d6a58618b043c989c.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/KhronosGroup-OpenCL-CLHPP-d62a02090625655e5b2d791d6a58618b043c989c.tar.gz
-- Using source at C:/src/vcpkg/buildtrees/opencl/src/8b043c989c-02b68d7dd1.clean
-- Downloading https://www.python.org/ftp/python/3.8.3/python-3.8.3-embed-amd64.zip...
-- Generating OpenCL C++ headers done
-- Downloading https://github.com/KhronosGroup/OpenCL-ICD-Loader/archive/e6e30ab9c7a61c171cf68d2e7f5c0ce28e2a4eae.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/KhronosGroup-OpenCL-ICD-Loader-e6e30ab9c7a61c171cf68d2e7f5c0ce28e2a4eae.tar.gz
-- Applying patch 0001-include-unistd-for-gete-ug-id.patch
-- Using source at C:/src/vcpkg/buildtrees/opencl/src/e28e2a4eae-7bb6371b0a.clean
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:106 (message):
    Command failed: ninja -v
    Working Directory: C:/src/vcpkg/buildtrees/opencl/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\src\vcpkg\buildtrees\opencl\config-x64-windows-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:312 (vcpkg_execute_required_process)
  ports/opencl/portfile.cmake:65 (vcpkg_configure_cmake)
  scripts/ports.cmake:135 (include)


Error: Building package opencl:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: opencl:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.

Enviado do Email<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: Phoebe<mailto:notifications@github.com>
Enviado:sexta-feira, 30 de outubro de 2020 05:20
Para: microsoft/vcpkg<mailto:vcpkg@noreply.github.com>
Cc:MrBolt2005<mailto:joao2ojf@hotmail.com>; Mention<mailto:mention@noreply.github.com>
Assunto: Re: [microsoft/vcpkg] [opencl[core]:x64-windows] build failure (#14114)


@MrBolt2005<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMrBolt2005&data=04%7C01%7C%7C4312744a0d524991ad9e08d87caca3c5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637396428202518258%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Ki0dzTkzbL%2BlxhEDrk1ble8fK2YP%2FwKSPSZjBaDi3ew%3D&reserved=0>, some suggest it might be caused by FAT32[1]; could you confirm that your D:\ drive is NTFS formatted? Reference from #1549 (comment)<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fvcpkg%2Fissues%2F1549%23issuecomment-318437627&data=04%7C01%7C%7C4312744a0d524991ad9e08d87caca3c5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637396428202528252%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=4FucrhOKqUGhjcT4j79N6gNe1xIRshywylOiq3ANSK4%3D&reserved=0>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fvcpkg%2Fissues%2F14114%23issuecomment-719402192&data=04%7C01%7C%7C4312744a0d524991ad9e08d87caca3c5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637396428202538246%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=cl1HqdooaaoLIZpI2ghe12GnA1%2B7fDUt7QHWH%2BMA0zw%3D&reserved=0>, or unsubscribe<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAQLMPXZUXZG5P2FGAVO6QC3SNJZMHANCNFSM4SWBN34A&data=04%7C01%7C%7C4312744a0d524991ad9e08d87caca3c5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637396428202548249%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=8HmiVOrvQ7ieoLba1zpNHsmIMWr4T38MJwDrsKAk2aQ%3D&reserved=0>.

[1/2] cmd /c "cd ../../x64-windows-dbg && "C:/Program Files/CMake/bin/cmake.exe" "C:/src/vcpkg/buildtrees/opencl/src/e28e2a4eae-7bb6371b0a.clean" "-DOPENCL_ICD_LOADER_HEADERS_DIR=C:/src/vcpkg/packages/opencl_x64-windows/include" "-DOPENCL_ICD_LOADER_REQUIRE_WDK=ON" "-DUSE_DYNAMIC_VCXX_RUNTIME=1" "-DCMAKE_MAKE_PROGRAM=C:/src/vcpkg/downloads/tools/ninja/1.10.1-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/src/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/src/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/src/vcpkg/packages/opencl_x64-windows/debug""
FAILED: ../../x64-windows-dbg/CMakeCache.txt
cmd /c "cd ../../x64-windows-dbg && "C:/Program Files/CMake/bin/cmake.exe" "C:/src/vcpkg/buildtrees/opencl/src/e28e2a4eae-7bb6371b0a.clean" "-DOPENCL_ICD_LOADER_HEADERS_DIR=C:/src/vcpkg/packages/opencl_x64-windows/include" "-DOPENCL_ICD_LOADER_REQUIRE_WDK=ON" "-DUSE_DYNAMIC_VCXX_RUNTIME=1" "-DCMAKE_MAKE_PROGRAM=C:/src/vcpkg/downloads/tools/ninja/1.10.1-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/src/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/src/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/src/vcpkg/packages/opencl_x64-windows/debug""
-- The C compiler identification is MSVC 19.27.29112.0
-- The CXX compiler identification is MSVC 19.27.29112.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE
-- Looking for secure_getenv
-- Looking for secure_getenv - not found
-- Looking for __secure_getenv
-- Looking for __secure_getenv - not found
CMake Error at CMakeLists.txt:100 (message):
  The Windows WDK was not found.  Consider disabling
  OPENCL_ICD_LOADER_REQUIRE_WDK.  Aborting.


-- Configuring incomplete, errors occurred!
See also "C:/src/vcpkg/buildtrees/opencl/x64-windows-dbg/CMakeFiles/CMakeOutput.log".
See also "C:/src/vcpkg/buildtrees/opencl/x64-windows-dbg/CMakeFiles/CMakeError.log".
[2/2] cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/src/vcpkg/buildtrees/opencl/src/e28e2a4eae-7bb6371b0a.clean" "-DOPENCL_ICD_LOADER_HEADERS_DIR=C:/src/vcpkg/packages/opencl_x64-windows/include" "-DOPENCL_ICD_LOADER_REQUIRE_WDK=ON" "-DUSE_DYNAMIC_VCXX_RUNTIME=1" "-DCMAKE_MAKE_PROGRAM=C:/src/vcpkg/downloads/tools/ninja/1.10.1-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/src/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/src/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/src/vcpkg/packages/opencl_x64-windows""
FAILED: ../CMakeCache.txt
cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/src/vcpkg/buildtrees/opencl/src/e28e2a4eae-7bb6371b0a.clean" "-DOPENCL_ICD_LOADER_HEADERS_DIR=C:/src/vcpkg/packages/opencl_x64-windows/include" "-DOPENCL_ICD_LOADER_REQUIRE_WDK=ON" "-DUSE_DYNAMIC_VCXX_RUNTIME=1" "-DCMAKE_MAKE_PROGRAM=C:/src/vcpkg/downloads/tools/ninja/1.10.1-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/src/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/src/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/src/vcpkg/packages/opencl_x64-windows""
-- The C compiler identification is MSVC 19.27.29112.0
-- The CXX compiler identification is MSVC 19.27.29112.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE
-- Looking for secure_getenv
-- Looking for secure_getenv - not found
-- Looking for __secure_getenv
-- Looking for __secure_getenv - not found
CMake Error at CMakeLists.txt:100 (message):
  The Windows WDK was not found.  Consider disabling
  OPENCL_ICD_LOADER_REQUIRE_WDK.  Aborting.


-- Configuring incomplete, errors occurred!
See also "C:/src/vcpkg/buildtrees/opencl/x64-windows-rel/CMakeFiles/CMakeOutput.log".
See also "C:/src/vcpkg/buildtrees/opencl/x64-windows-rel/CMakeFiles/CMakeError.log".
ninja: build stopped: subcommand failed.

Enviado do Email<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: Phoebe<mailto:notifications@github.com>
Enviado:domingo, 1 de novembro de 2020 22:50
Para: microsoft/vcpkg<mailto:vcpkg@noreply.github.com>
Cc:MrBolt2005<mailto:joao2ojf@hotmail.com>; Mention<mailto:mention@noreply.github.com>
Assunto: Re: [microsoft/vcpkg] [opencl[core]:x64-windows] build failure (#14114)


Could you attach failure log C:\src\vcpkg\buildtrees\opencl\config-x64-windows-out.log?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fvcpkg%2Fissues%2F14114%23issuecomment-720194382&data=04%7C01%7C%7Cfcb30569761040cd903a08d87ed1a34b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637398786132011461%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ZQMtgoVLBe5VEVj7pWEI5P7eewsPg5cUhAry0J8cPX4%3D&reserved=0>, or unsubscribe<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAQLMPX4IMWCFG45WYTHAQATSNYF5JANCNFSM4SWBN34A&data=04%7C01%7C%7Cfcb30569761040cd903a08d87ed1a34b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637398786132021456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=hVtpbTR0uXBlhZfz%2B6so13nSmv2d3poRMsCbrPz6IUk%3D&reserved=0>.

@MrBolt2005, could you install windows WDK and try again? https://docs.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk

Failure:
CMake Error at CMakeLists.txt:100 (message):
  The Windows WDK was not found.  Consider disabling
  OPENCL_ICD_LOADER_REQUIRE_WDK.  Aborting.
…

It’s already installed, and the PATH is already updated.

Enviado do Email<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: Phoebe<mailto:notifications@github.com>
Enviado:segunda-feira, 2 de novembro de 2020 03:20
Para: microsoft/vcpkg<mailto:vcpkg@noreply.github.com>
Cc:MrBolt2005<mailto:joao2ojf@hotmail.com>; Mention<mailto:mention@noreply.github.com>
Assunto: Re: [microsoft/vcpkg] [opencl[core]:x64-windows] build failure (#14114)


@MrBolt2005<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMrBolt2005&data=04%7C01%7C%7Ccea24e0f909a47c73e0808d87ef761b7%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637398948239701021%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=VRlUyVDxv%2BIWn8kAi58SE%2BtqC%2FQ%2FvKh4i3I3fzJqjtw%3D&reserved=0>, could you install windows WDK and try again? https://docs.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fwindows-hardware%2Fdrivers%2Fdownload-the-wdk&data=04%7C01%7C%7Ccea24e0f909a47c73e0808d87ef761b7%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637398948239701021%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=qXm8c%2B9w8fufIfhXZnQXTsNyf497hsQZm5qvuFGI59M%3D&reserved=0>

Failure:
CMake Error at CMakeLists.txt:100 (message):
The Windows WDK was not found. Consider disabling
OPENCL_ICD_LOADER_REQUIRE_WDK. Aborting.
…

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fvcpkg%2Fissues%2F14114%23issuecomment-720262791&data=04%7C01%7C%7Ccea24e0f909a47c73e0808d87ef761b7%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637398948239711013%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=TVGU1vZodrE4KiOxatcFYEuQaYrJItr4FrWE5wpMUqk%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAQLMPXYZRFDUI466D33BYTTSNZFSPANCNFSM4SWBN34A&data=04%7C01%7C%7Ccea24e0f909a47c73e0808d87ef761b7%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637398948239711013%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=gKvs9YNvjB8DM6epsIwc1g4Y77qAy8%2BPPYPlMFL%2FLQQ%3D&reserved=0>.

--- a/ports/opencl/CONTROL
+++ b/ports/opencl/CONTROL
@@ -1,8 +1,5 @@
 Source: opencl
 Version: 2.2
-Port-Version: 4
+Port-Version: 5
 Homepage: https://github.com/KhronosGroup/OpenCL-Headers
 Description: C/C++ headers and ICD loader (Installable Client Driver) for OpenCL
-
-Feature: wdk
-Description: Windows Driver Kit support

--- a/ports/opencl/portfile.cmake
+++ b/ports/opencl/portfile.cmake
@@ -1,12 +1,3 @@
-if("wdk" IN_LIST FEATURES)
-    if(NOT VCPKG_TARGET_IS_WINDOWS)
-        message(FATAL_ERROR "Windows Driver Kit support is only available builds targeting Windows")
-    endif()
-    set(WITH_WDK ON)
-else()
-    set(WITH_WDK OFF)
-endif()
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO KhronosGroup/OpenCL-Headers
@@ -15,11 +6,7 @@ vcpkg_from_github(
     HEAD_REF master
 )
 
-file(INSTALL
-        "${SOURCE_PATH}/CL"
-    DESTINATION
-        ${CURRENT_PACKAGES_DIR}/include
-)
+file(INSTALL "${SOURCE_PATH}/CL" DESTINATION ${CURRENT_PACKAGES_DIR}/include)
 
 # OpenCL C++ headers
 vcpkg_from_github(
@@ -67,7 +54,7 @@ vcpkg_configure_cmake(
     PREFER_NINJA
     OPTIONS
         -DOPENCL_ICD_LOADER_HEADERS_DIR=${CURRENT_PACKAGES_DIR}/include
-        -DOPENCL_ICD_LOADER_REQUIRE_WDK=${WITH_WDK}
+        -DOPENCL_ICD_LOADER_REQUIRE_WDK=OFF
         -DUSE_DYNAMIC_VCXX_RUNTIME=${USE_DYNAMIC_VCXX_RUNTIME}
 )
 

